### PR TITLE
Admin dashboard

### PR DESCRIPTION
**Pull request by agriffard**
Use a jumbotron.

Rename title.

Load shapes starting with `AdminDashboardCard` from modules:

image

### CODE REVIEW

**Comment by agriffard** [comment]
image

**Comment by hishamco** [comment]
I'm still have a doubt using color variations in buttone, it would be nice to use one color, frankly there's no mean to use different color here

Overall looks good, hope it's mobile friendly which I think

**Comment by hishamco** [comment]
Please make sure it's 100% fine with the dark theme

**Comment by deanmarcussen** [comment]
Hmm, what will happen if any of these modules are disabled?

Hint: this is why we have drivers

**Comment by hishamco** [comment]
Good point Dean, what if we follow certain convensions like what we did in `AdminMenu`, so every module could participate on dashboard. So whenever a module has a view named `Dashboard.chtml` it could be placed into a dashboard module, that we we could fix the issue that Dean mentioned above

**Comment by agriffard** [comment]
Ok, keeping it minimal for the moment.

**Comment by hishamco** [comment]
We could keep the cards as its, then we could ability to load them dynamically

**Comment by agriffard** [comment]
Did an example in Users with the count retrieved from a Query.

We could also display the N latest modified items or some stats.

Will try to take the permissions into account before displaying a card or a link.

**Comment by Skrypt** [comment]
Maybe remove the title and make it part of the title in the jumbotron

screencapture-localhost-5001-Admin-2020-12-06-17_56_12

**Comment by hishamco** [comment]
The first jumbotron is good looking

**Comment by agriffard** [comment]
Title removed.
Didn't change the padding or it should be done in a specific css for the admin jumbotron.

Kept the word site:

image

Dashboard is a bit to long on small screens:

image

Compared to:

image

File: src/OrchardCore.Modules/OrchardCore.Admin/Views/AdminDashboard.cshtml [hunk 1]
```diff
@@ -1,4 +1,23 @@
-<div class="alert alert-success">
-    <h4>@T["Welcome to Orchard"]</h4>
-    <p>@T["Feel free to browse the menu and discover all its possibilities."]</p>
+@using OrchardCore.DisplayManagement;
+@inject OrchardCore.DisplayManagement.Descriptors.IShapeTableManager ShapeTableManager
+
+@{
+    var shapeTable = ShapeTableManager.GetShapeTable("TheAdmin");
```
Comment (deanmarcussen) [inline src/OrchardCore.Modules/OrchardCore.Admin/Views/AdminDashboard.cshtml:5]: I would suggest this is done with Drivers, and a 
`IDisplayManager<AdminDashboard>()` (making an identifier class called `AdminDashboard` in `Admin.Abstractions` to drive the display with)

Here this code will fail if `GetShapeTable("TheAdmin")` a different admin theme was used.

If you do it with drivers, then you get placement - the ability to hide them / move them around. All the cool things that modular display management does.

**Review by Skrypt — APPROVED** [review]

**Comment by agriffard** [comment]
Demoed during the meeting.

Suggestions:
- Make it a feature?
- Allow to add and organize Widgets by drag and drop and store states like in a zone of a layer.

**Comment by agriffard** [comment]
Initial issue: #2246

**Comment by hishamco** [comment]
Can we merge this while it's not break anything, then you can continue on widgets and make it feature as Seb suggested in the standup

**Comment by agriffard** [comment]
No, it was just a POC.

We made a design meeting and the implementation will be completely different (and more powerful hopefully):
https://github.com/OrchardCMS/OrchardCore/issues/2246#issuecomment-742687838

Closing this one.
Replaced by https://github.com/OrchardCMS/OrchardCore/pull/7943

**Comment by weedkiller** [comment]
@agriffard I saw you closed this, please share info/sample on creating charts/d3 visuals/tables/calendar, with UI pattern to wire data source

Also please help ensure they responsive if we have SVG element inside.

**Comment by fasteddys** [comment]
Can you also add a graph or chart / calendar or some non text widget here

**Pull request closed without merging**